products-and-applications/features.md

```diff
@@ -1,9 +1,32 @@
 ---
-layout: coming_soon
-nav_order: 15
-parent: Products
+nav_order: 12
 title: Features
```

Comment:
Maybe this belongs under an `apis/` directory or we just make a `/features` directory?